méli-mélo/th-zev/zev.html

```diff
@@ -0,0 +1,210 @@
+          <li>Use <code>.bg-zev-purple</code> in combination with <code>.text-white</code> to ensure sufficient contrast between text and background</li>
+          <li>In the case of <code>.bg-zev-purple</code> use a fallback colour such as <code>.bg-dark</code> for sections and paragraphs, <code>.btn-info</code> for buttons or <code>.panel-default</code> for panels to accommodate progressive enhancement</li>
+          <li>Wrap text in a <code>&lt;span&gt;</code> tag if a style does not appear as it is meant to, due to specificity reasons</li>
+          <li>For buttons using these colours ensure a fallback is provided using the <code>.btn-default</code> class</li>
+        </ul>
+      </div>
+      <div class="col-md-6">
+        <h3 class="text-danger"><span class="glyphicon glyphicon-remove-circle"></span> Incorrect use</h3>
+        <p>Compliance point(s):</p>
+        <ul>
+          <li>Do not use this component in a way that conflicts with the preceding compliance <span class="nowrap">point(s)</span></li>
+          <li>Do not use as the only way to communicate information or intent, as colour alone is not accessible</li>
+          <li>These colours should not be used to convey meaning unless there is also text to support the distinction. Remember that these classes may be removed and some users will not see them.</li>
+        </ul>
+      </div>
+    </div>
+
+    <h3>CSS classes</h3>
+    <dl class="dl-horizontal">
+      <dt><code>.bg-zev-green</code></dt>
+      <dd>Set a background colour in an element or well to green</dd>
+      <dt><code>.bg-zev-purple</code></dt>
+      <dd>Set a background colour in an element or well to purple</dd>
+      <dt><code>.btn-zev-green</code></dt>
+      <dd>Set a button to the ZEV purple green</dd>
+      <dt><code>.btn-zev-purple</code></dt>
+      <dd>Set a button to the ZEV purple style</dd>
+      <dt><code>.panel-zev-green</code></dt>
+      <dd>Set a Panel to the ZEV purple green header, footer, and border.</dd>
+      <dt><code>.panel-zev-purple</code></dt>
+      <dd>Set a Panel to the ZEV purple purple header, footer, and border.</dd>
+    </dl>
+
+    <h2>Examples</h2>
+    <h3 id="a2">Well example</h3>
+
+    <div class="well mrgn-bttm-0 text-center bg-zev-green">
+      <h4 class="well-text">This is a <code>.bg-zev-green</code> well wrapper with <code>.well-text</code> on the header</h4>
+      <p><a class="btn btn-lg btn-default mrgn-tp-md btn-zev-purple" href="#">Station locator</a></p>
+    </div>
+    <div class="well bg-dark text-white bg-zev-purple">
+      <ul class="list-unstyled lst-spcd mrgn-lft-md mrgn-rght-sm mrgn-bttm-0 text-white">      
+        <li><h4 class="text-white mrgn-tp-sm well-text">This is a <code>.bg-zev-purple</code> well</h4></li>
+        <li><a class="text-white well-link" href="#">Incentives for buyers and dealerships</a></li>
+        <li><a class="text-white well-link" href="#">Infrastructure and awareness project funding</a></li>
+      </ul>    
```

Comment:
```suggestion
      <h4 class="text-white mrgn-tp-sm well-text">This is a <code>.bg-zev-purple</code> well</h4>
      <ul class="list-unstyled lst-spcd mrgn-lft-md mrgn-rght-sm mrgn-bttm-0 text-white">
        <li><a class="text-white well-link" href="#">Incentives for buyers and dealerships</a></li>
        <li><a class="text-white well-link" href="#">Infrastructure and awareness project funding</a></li>
      </ul>    
```

Is this heading level 4 required here in the `<li>`? From the structure aspect of this working example page it create a confusion for the sibling `<li>` element